ground/gcs/src/plugins/uavsettingsimportexport/importsummary.cpp

```diff
@@ -54,6 +91,15 @@ ImportSummaryDialog::ImportSummaryDialog( QWidget *parent) :
 ImportSummaryDialog::~ImportSummaryDialog()
 {
     delete ui;
+    delete importedObjects;
```

Comment:
This doesn't look like it needs deleting. In addition, there are a bunch of `new` commands in this PR, and it looks like they could in general be removed/refactored.